comp/core/tagger/component.go

```diff
@@ -43,8 +43,8 @@ type Component interface {
 	Standard(entityID string) ([]string, error)
 	List() types.TaggerListResponse
 	GetEntity(entityID string) (*types.Entity, error)
-	Subscribe(cardinality types.TagCardinality) chan []types.EntityEvent
-	Unsubscribe(ch chan []types.EntityEvent)
```

Comment:
A subscriber is now identified by a unique subscription id, and not by a channel. 

Ofcourse each subscriber still has a unique channel, the subscription data is organised by mapping a subscription id to its related information. 

We can still use the channel to identify the subscriber, but it will require an extra mapping step.